.github/actions/upload-test-artifacts/action.yml

```diff
@@ -55,6 +69,15 @@ runs:
         # -ir => recursive include all files in pattern
         7z a "test-reports-$Env:FILE_SUFFIX.zip" -ir'!test\*.xml'
 
+    - name: Zip usage log for upload
+      if: runner.os == 'Windows' && !inputs.use-gha
+      shell: powershell
+      env:
+        FILE_SUFFIX: ${{ inputs.file-suffix }}
+      run: |
```

Comment:
Do we also need to remove any existing `usage-log-*.zip` here?

Reply:
based on the other windows zips, it doesnt look like you need to